c2corg_ui/static/js/style/autoheight.js

```diff
@@ -0,0 +1,66 @@
+goog.provide('app.autoHeightDirective');
+goog.provide('app.AutoHeightController');
+
+goog.require('app');
+
+
+/**
+ * Apply automatic height depending on text to all textarea tags
```

Comment:
> I am not that sure it is really relevant that the textarea height grows with no limit

there is a limit! a limit that i defined 'à l'oeil' and it's max 90% of the viewport height. See https://github.com/c2corg/v6_ui/pull/1232/commits/5d1e0cecb082acd59e2aa51c904c458f564b2b38#diff-bca22f81eb4368fbe89d6009bfa48f2cR58 

Indeed working with a very long text was annoying when the textarea expanded to the whole lenght. 

I think it's hard to imagine the way it works and say 'i think that...' without actually testing it, because it's a little bit abstract. I suggest you test it - it works pretty well in my opinion! on mobile and on desktop.